tests/imodels-access-backend-tests/src/IModelsAccessBackendTestsConfig.ts

```diff
@@ -0,0 +1,82 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import * as dotenv from "dotenv";
+import { injectable } from "inversify";
+import { ApisConfigValues, AuthConfigValues, BaseIntegrationTestsConfig, TestSetupError, TestUsersConfigValues } from "@itwin/imodels-client-test-utils";
+
+@injectable()
+export class IModelsAccessBackendTestsConfig implements BaseIntegrationTestsConfig {
+  public readonly testProjectName: string;
+  public readonly testIModelName: string;
+  public readonly auth: AuthConfigValues;
+  public readonly apis: ApisConfigValues;
+  public readonly testUsers: TestUsersConfigValues;
+
+  constructor() {
+    dotenv.config();
+    this.validateAllValuesPresent();
+
+    this.testProjectName = process.env.TEST_PROJECT_NAME!;
```

Comment:
I agree that it is pretty verbose, especially since the the config resolution is present in all test packages. Even though it is a lot of code it is boilerplate - the loading of env variables is pretty simple so I think dotenv suffices. Yargs may also be a good solution but I don't think we need to make a switch now.